Sources/Afluent/Workers/ShareFromCache.swift

```diff
@@ -8,6 +8,8 @@
 import Foundation
 
 extension AsynchronousUnitOfWork {
+    private typealias CachedWork = Workers.Share<Workers.HandleEvents<Self, Success>, Success>
```

Comment:
Still necessary?

Reply:
We're still using it on L41 for the optional cast. Do you think it's fine to just not cast at all? I mean, presumably the hash key should do the differentiating for us, but curious what you think.

Reply:
Oh yeah, I don't think we need the cast. At least I don't see any obvious reason why it grants extra safety.

Reply:
Ok, removed!